R/qenv-get_code.R

```diff
@@ -3,6 +3,7 @@
 #' @name get_code
 #' @param object (`qenv`)
 #' @param deparse (`logical(1)`) if the returned code should be converted to character.
+#' @param names (`character(n)`) if provided, returns the code only for objects specified in `names`.
```

Comment:
Food for thought: is it a better API to have this argument or to have a separate function, say `get_object_code`?

Reply:
we can also have `get_code()` that extracts a list for all the objects, and you could call `get_code()['object_name']`, unsure what is the best way in here yet